### DISCUSSION TITLE
How to convert json object to string [Duplicate?]

### DISCUSSION THREAD

**Discussion by GuckTubeYT** [nlohmann/json#2473] [General]
How to convert json object to string?
when i cout like this
`json j;
j["hello"] = "hi";
cout << j << endl;`
its work, but when i do string like this
`json j;
j["hello"] = "hi";
string je = j;
cout << je << endl;`
i got an error message =  [json.exception.type_error.302] type must be string, but is object

**Comment by nlohmann** [comment]
If you want a serialization, call `j.dump()` which returns a string. Otherwise, the exception tells you the problem: you cannot convert an object to a string.

**Comment by GuckTubeYT** [comment]
its work, thanks
